common/types.d.ts

```diff
@@ -127,6 +128,12 @@ declare global {
 
   type IConnectionKey = keyof IConnection
 
+  interface IosInfo {
```

Comment:
Interfaces should have I prefix:

```
interface OSInfo {
```

Reply:
I'm not sure what you are saying.  it does have an I prefix.

Reply:
Should not have an "I" prefix sorry, was a typo